sci-physics/rivet/rivet-3.1.10.ebuild

```diff
@@ -26,7 +26,8 @@ fi
 
 LICENSE="GPL-3+"
 SLOT="3"
-IUSE="+hepmc3 hepmc2 +zlib +python"
+IUSE="test +hepmc3 hepmc2 +zlib +python"
+RESTRICT="!test? ( test )"
```

Comment:
Since we are not adding any test dependencies or test specific configuration. We don't actually need ` test` USE flag here.